ztp/kube-compare-reference/optional/image-registry/ImageRegistryConfig.yaml

```diff
@@ -5,11 +5,15 @@ metadata:
 spec:
   logLevel: Normal
   managementState: Managed
+  {{- if .spec.observedConfig }}
   observedConfig: {{ .spec.observedConfig | toYaml | indent 2 }}
```

Comment:
Not sure how this CR looks like, just making sure it should be `indent` instead of newline with `nindent`

Reply:
That's a good question.  In the one example I have it's just `null`, so shouldn't need `indent` at all.

Reply:
Reading more about it [in the api docs](https://docs.openshift.com/container-platform/4.16/rest_api/operator_apis/config-imageregistry-operator-openshift-io-v1.html#spec) didn't help much...

I think it looked like `observedConfig` should just be passed-through directly, whereas `unsupportedConfigOverrides` should be actively prohibited.